src/mongocxx/write_concern.hpp

```diff
@@ -183,7 +204,14 @@ class MONGOCXX_API write_concern {
     ///
     /// @return The number of required nodes.
     ///
-    std::int32_t nodes() const;
+    stdx::optional<std::int32_t> nodes() const;
```

Comment:
For this method, would an unset optional mean that no write concern level had been set, or that no nodes are required?